meta-resin-common/classes/image_types_resin.bbclass

```diff
@@ -60,6 +60,9 @@ RESIN_DATA_FS_LABEL ?= "resin-data"
 # Used as extension for fingerprint files
 FINGERPRINT_EXT ?= "fingerprint"
 
+# Encode of the config file
+CONFIG_FILE_ENC ?= "json"
```

Comment:
Why do You specify the extension as a separate variable? You always write {} in it.